.github/workflows/run_checks.yaml

```diff
@@ -190,7 +188,7 @@ jobs:
         run: |
           printf -- "---\ntitle: Apply Patch Status ${{ inputs.patch_name }}\n---\n" > issue.md
           printf "\nFailed to apply patch\n" >> issue.md
-          cat out >> issue.md
+          cat out_base out_tot >> issue.md || true
```

Comment:
Remove the || true since we can safely assume that both out_base and out_tot exist